modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -35,9 +35,6 @@ support with your installation logs.
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
 * Create and configure networking and load balancers in {cp}.
 * Create control plane and compute roles.
```

Comment:
Thinking this whole block can go too

Reply:
This is the first instance of 'Create and configure networking and load balancers' and 'Create control plane and compute roles' in the assembly, so I left them in here and removed them from later modules.